js_task1/firstTask.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS tasks</title>
+  </head>
+  <body>
+    <section class="first-task">
+      <a href="./index.html"> Back </a>
+      <h1>First Task</h1>
+      <input type="text" name="" id="" value="10" />
```

Comment:
It is better to set value for 'name' but id you can remove.

Reply:
done